src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs

```diff
@@ -10,21 +10,21 @@ namespace Microsoft.Framework.PackageManager.List
 {
     public class LibraryDependencyFlatRenderer
     {
-        private IReport _output;
+        private readonly IReport _output;
+        private readonly bool _hideDependent;
 
-        public LibraryDependencyFlatRenderer(IReport output)
+        public LibraryDependencyFlatRenderer(IReport output, bool hideDependent)
         {
             _output = output;
+            _hideDependent = hideDependent;
         }
 
         public void Render(IGraphNode<Library> root)
         {
             var dict = FindImmediateDependent(root);
-            foreach (var key in dict.Keys.OrderBy(library => library.Name))
-            {
-                _output.WriteLine(key.ToString().White().Bold());
-                _output.WriteLine("    -> {0}", string.Join(", ", dict[key].Select(lib => lib.ToString()).OrderBy(s => s)));
-            }
+
+            RenderLibraries(dict.Keys.Where(library => library.IsGacOrFrameworkReference), dict);
```

Comment:
Can we optimize this a bit by storing the OrderedKeys on a local variable and do the where check - RenderLibraries can remove OrderBy as well.